src/input.rs

```diff
@@ -144,10 +144,11 @@ impl<'a> Input<'a> {
 				#[cfg(feature = "zeroize")]
 				{
 					// `Zeroize(crate = "..")` is used.
-					if let DeriveTrait::Zeroize {
-						crate_: Some(_), ..
-					} = **trait_
-					{
+					if let DeriveTrait::Zeroize { crate_: Some(_) } = **trait_ {
+						continue;
+					}
+					// `ZeroizeOnDrop(crate = "..")` is used.
+					if let DeriveTrait::ZeroizeOnDrop { crate_: Some(_) } = **trait_ {
 						continue;
 					}
```

Comment:
wouldn't that fit in a single match statement `Zeroize | ZeroizeOnDrop` ?

Reply:
Indeed! On it.

Reply:
Turned out to be not much prettier, used an `if` because I only needed one pattern.